collection.c

```diff
@@ -609,7 +611,7 @@ static int send_message(zval *this_ptr, mongo_connection *connection, mongo_buff
 		return 0;
 	}
 
-	if (is_gle_op(this_ptr, options, &link->servers->options TSRMLS_CC)) {
+	if (is_gle_op(this_ptr, options, &link->servers->options, 0 TSRMLS_CC)) {
```

Comment:
I'm not keen on arbitrary values - perhaps define a constant for SILENT and NOISY? Or perhaps renaming the argument to "show_warnings", to also get rid of the ! in front of silent (as IMO, that reads harder).